src/state/apis/zapper/zapperApi.ts

```diff
@@ -87,35 +87,26 @@ export const zapperApi = createApi({
         const { data: res } = await axios.request({ ...payload })
         const zapperV2AppTokensData = V2AppTokensResponse.parse(res)
 
-        const parsedData = zapperV2AppTokensData.reduce<GetZapperAppTokensOutput>(
-          (acc, appTokenData) => {
-            const chainId = zapperNetworkToChainId(appTokenData.network)
-            if (!chainId) return acc
-            const assetId = toAssetId({
-              chainId,
-              assetNamespace: 'erc20', // TODO: bep20
-              assetReference: appTokenData.address,
-            })
-
-            acc[assetId] = appTokenData
-            return acc
-          },
-          {},
-        )
-
         const assets = selectAssets(getState() as ReduxState)
-        const zapperAssets = zapperV2AppTokensData.reduce<AssetsState>(
+
+        const { assets: zapperAssets, data } = zapperV2AppTokensData.reduce<{
```

Comment:
single reduce, was in here thinking we were storing nft assets and changed, but can remove since not related at all

Reply:
Either or, totally agnostic with it, the single reduce makes this more performant, though parsing the data and making the assets upsertion payload was more declarative IMO - happy with both tradeoffs but would be inclined to keeping this more performant version

Reply:
will leave it since it's there then